sound/soc/sof/sof-audio.h

```diff
@@ -290,6 +292,8 @@ struct snd_sof_pcm_stream {
 	u64 stream_start_offset;
 	struct sof_ipc4_fw_registers *fw_reg;
 	u32 llp_offset;
+	u32 last_pos;
+	u32 wrap_time;
```

Comment:
not following the definition of a time as a u32?

Reply:
sorry, it is misleading. it should be wrap_count. I often take time as count or number